extension/chrome/dev/smime.ts

```diff
@@ -0,0 +1,44 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
+
+'use strict';
+
+import { Catch } from '../../js/common/platform/catch.js';
+
+import * as forge from 'node-forge';
+
+const HEADERS = `MIME-Version: 1.0
+Content-Type: application/pkcs7-mime; name="smime.p7m"; smime-type=enveloped-data
+Content-Transfer-Encoding: base64
+Content-Disposition: attachment; filename="smime.p7m"
+Content-Description: S/MIME Encrypted Message`;
+
+Catch.try(async () => {
+  function wrap(string: string) {
+    var lines = [];
+    var index = 0;
+    var length = string.length;
+    while (index < length) {
+      lines.push(string.slice(index, index += 76));
```

Comment:
I'm a little ahead with this comment for this particular level of PoC, but you can use
```ts
(window as unknown as BrowserWindow)['emailjs-mime-codec'].foldLines(newMsg.plaintext, 76, true)
```
Take a look at `signed-msg-mail-formatter`.